edb/server/protocol/binary.pyx

```diff
@@ -465,9 +465,9 @@ cdef class EdgeConnection(frontend.FrontendConnection):
                 f'authentication failed: malformed claims section in JWT'
             ) from None
 
-        self._check_jwt_authz(claims, token_version, user)
+        self._check_jwt_authz(claims, token_version, user, params['database'])
```

Comment:
Let's pass `database` as an explicit argument along with `user` to avoid this sort of regression in the auth call stack.